test/loader.spec.js

```diff
@@ -208,20 +211,20 @@ describe('loader', () => {
 					{}
 				)
 			);
+		});
 
-			it(
-				'should configure emitCss=true',
-				testLoader(
-					'test/fixtures/css.html',
-					function(err, code, map) {
-						expect(err).not.to.exist;
+		it(
+			'should configure emitCss=true',
```

Comment:
was this moved outside the above function on purpose? does it still work if we put it back where it was originally located?

Reply:
> was this moved outside the above function on purpose? does it still work if we put it back where it was originally located?

no, fixed it.